src/assets/stylesheets/shared/_overrides-for-patternfly.less

```diff
@@ -38,3 +38,7 @@ h5 {
 .dropdown {
   a:hover { text-decoration: none; }
 }
+.dropdown-menu {
+  max-height: 230px;
+  overflow-y: scroll;
```

Comment:
Can we put ```overflow-y: auto```? Because scroll property always shows scroll bar space even if there is less element.